src/components/access_group/AccessGroupInformationEdit.vue

```diff
@@ -0,0 +1,359 @@
+          </p>
+        </div>
+        <div class="content-area__row close-group-container">
+          <input type="checkbox" v-model="closeGroupConfirmed" />
+          <label class="content-area__label">{{fluent('access-group_close-group', 'confirm-text')}}</label>
+          <Button
+            :disabled="!closeGroupConfirmed"
+            class="button--primary primary-actdion"
+            @click="handleCloseGroupClicked"
+          >{{fluent('access-group_close-group', 'confirm-close')}}</Button>
+        </div>
+      </template>
+    </AccessGroupEditPanel>
+  </section>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex';
+import TextInput from '@/components/ui/TextInput.vue';
+import TextArea from '@/components/ui/TextArea.vue';
+import Button from '@/components/ui/Button.vue';
+import Icon from '@/components/ui/Icon.vue';
+import AccessGroupEditPanel from '@/components/access_group/AccessGroupEditPanel.vue';
+import AccessGroupMarkdownGuide from '@/components/access_group/AccessGroupMarkdownGuide.vue';
+
+export default {
+  name: 'AccessGroupInformationEdit',
+  components: {
+    TextInput,
+    TextArea,
+    Button,
+    Icon,
+    AccessGroupEditPanel,
+    AccessGroupMarkdownGuide,
+  },
+  props: [],
+  mounted() {},
+  data() {
+    const accessGroup = this.$store.getters['accessGroup/getGroup'];
+    const terms = this.$store.getters['accessGroup/getTerms'];
+    return {
+      groupDescriptionData: accessGroup.description,
+      groupDescriptionDirty: false,
+      groupTermsData: terms,
+      groupTermsRequiredData: accessGroup.terms,
+      groupTermsDirty: false,
+      groupTypeData: accessGroup.type,
+      groupTypeDirty: false,
+      closeGroupConfirmed: false,
+    };
+  },
+  watch: {
+    groupDescriptionData(val) {
+      this.groupDescriptionDirty = true;
+    },
+    groupTermsData(val) {
+      this.groupTermsDirty = true;
+    },
+    groupTypeData(val) {
+      this.groupTypeDirty = true;
+    },
+    groupTermsRequiredData(val) {
+      this.groupTermsDirty = true;
+    },
+  },
+  methods: {
+    ...mapActions({
+      updateGroup: 'accessGroup/updateGroup',
+      deleteTerms: 'accessGroup/deleteTerms',
+      updateTerms: 'accessGroup/updateTerms',
+      addTerms: 'accessGroup/addTerms',
+      closeGroup: 'accessGroup/closeGroup',
+    }),
+    handleDescriptionUpdateClicked(e) {
+      e.preventDefault();
+      this.updateGroup({
+        field: 'description',
+        value: this.groupDescriptionData,
+      }).then(() => {
+        this.groupDescriptionDirty = false;
+        this.$root.$emit('toast', {
+          content: 'Group description updated',
+        });
+      });
+      return false;
+    },
+    handleTypeUpdateClicked() {
+      this.updateGroup({ field: 'type', value: this.groupTypeData }).then(
+        () => {
+          this.groupTypeDirty = false;
+          this.$root.$emit('toast', {
+            content: 'Group type updated',
```

Comment:
Should these maybe be set as variables, eg, `GROUP_TYPE_UPDATED_MSG`?

Reply:
I think so...and as I think about it more, this whole thing should probably be in a mixin so deep components don't have to go to root emit.

Reply:
So rather than being in variables, they should have been in fluent. I've updated this to be the case.